code/game/objects/items/cigs_lighters.dm

```diff
@@ -723,8 +723,23 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	. = ..()
 	if(!overlay_state)
 		overlay_state = pick(overlay_list)
+	AddComponent(\
+		/datum/component/bullet_intercepting,\
+		block_chance = 0.5,\
+		active_slots = ITEM_SLOT_SUITSTORE,\
+		on_intercepted = CALLBACK(src, PROC_REF(on_intercepted_bullet)),\
+	)
 	update_appearance()
 
+/// Destroy the lighter when it's shot by a bullet
```

Comment:
wish we could make the lighter have a little pockmark and qdel the component but that's for another time i suppose